fmskill/compare.py

```diff
@@ -389,6 +402,119 @@ def _parse_by(self, by, n_models, n_obs):
             raise ValueError("Invalid by argument. Must be string or list of strings.")
         return by
 
+    def spatial_skill(
+        self,
+        bins=5,
+        binsize: float = None,
+        retbins: bool = False,
```

Comment:
It seems that the retbins argument is not used?

Reply:
No, I did not end up using it. Idea was to pass this to pd.cut: https://pandas.pydata.org/docs/reference/api/pandas.cut.html
But I am not sure if there is a scenario where it is of interest to return the resulting bins from pd.cut().
Should we just get rid of it?